cypress/integration/common/i_fill_out.js

```diff
@@ -51,6 +51,14 @@ When("I force fill out", (dataTable) => {
             });
 });
 
+Then("I fill out {string} element with {string}", (element, value) => {
+   cy.get(element).clear({ force: true }).type(value, { force: true })
+});
+
```

Comment:
Since this uses force, would you be willing to explicitly say "I force fill out" here? That is the pattern I did, to make it clear in the English text when we are using force and when not. Also, notice at the top of this file there is a step called "I type {string} into old style id {string}", which seems to me like a sibling to this but without the force. So maybe these steps could be named similarly?  To be honest I don't really like "old style id" and wished I called it something else. If you like I could rename that in a separate PR though.